framework/web/AssetBundle.php

```diff
@@ -59,6 +59,14 @@ class AssetBundle extends Object
      */
     public $baseUrl;
     /**
+     * @var boolean|string the URI scheme to use. Possible values:
+     *
+     * - `false` (default): assets URL without host info.
+     * - `true`: assets URL with an absolute base URL whose scheme is the same as that in [[\yii\web\UrlManager::hostInfo]].
+     * - string: assets URL with an absolute base URL with the specified scheme (either `http` or `https`).
+     */
+    public $scheme = false;
```

Comment:
Also it seems description is wrong i.e. `false` means "use scheme as specified in a asset manager".